### PR TITLE
Build using CUDA 10.1

### LINKED ISSUE #719: Build Addons with CUDA 10.1
TF 2.1 will be compiled with CUDA 10.1 by default:
https://github.com/tensorflow/tensorflow/issues/26289

Our current docker container supports only CUDA 10 so we'll want to upgrade at some point to match the environment. I'm not sure if this is an immediate breakage though as it looks like the nightlies are still compatible:
https://colab.research.google.com/drive/10Be-m2cRDRH-_l5qjSxRRFbpZsgdFV-I

@yifeif how would you like to track updating the build env? Should we create an issue in custom-op?

### PR DESCRIPTION
**Pull request by seanpmorgan**
Closes #719 and #498 

Also I will be posting a more in depth bld refactor issue later

### CODE REVIEW

**Comment by seanpmorgan** [comment]
@tensorflow/sig-addons-maintainers ready for review.

File: README.md [hunk 1]
```diff
@@ -67,22 +67,22 @@ pip install tfa-nightly
 
 #### Installing from Source
 You can also install from source. This requires the [Bazel](
-https://bazel.build/) build system.
+https://bazel.build/) build system (version >= 1.0.0)
```
Comment (WindQAQ) [inline README.md]: miss a period.

Comment (seanpmorgan) [reply README.md]: Done. Thanks!

**Comment by WindQAQ** [comment]
@seanpmorgan It seems that CI build on GCP still runs with bazel==0.29.1.

https://source.cloud.google.com/results/invocations/ebcf1846-09a6-4058-a3ce-d9055a0127cf/log

**Pull request merged**